docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -499,3 +499,9 @@ the GPUs used for the experiment to be evenly distributed among the compute node
 -  A job may fail with the message ``resources failed with non-zero exit code``, Determined reports
    the exit code in the experiment logs. For example, the experiment logs contain ``srun: error:
    node002: task 0: Exited with exit code 7``.
+
+-  The CLI commands ``det slot enable…`` & ``det slot disable…`` are not supported in the HPC
+   cluster environment. Use of these commands will print an error message.
+
+-  In the HPC cluster environment, the ``det slot list`` command will not display the name of any
```

Comment:
```suggestion
-  In an HPC cluster environment, ``det slot list`` will not display the name of any
```

Reply:
Probably an drop 'In an HPC cluster environment' from both of these, since this is in the 'slurm known issues' documentation